app/src/marketplace/components/ProductPage/ProductOverview/index.jsx

```diff
@@ -0,0 +1,180 @@
+                            <div className={styles.statValue}>
+                                {stats.totalEarnings}
+                                <span className={styles.currency}> DATA</span>
+                            </div>
+                        </div>
+                        <div>
+                            <div className={styles.statHeading}>Active Members</div>
+                            <div className={styles.statValue}>{stats.memberCount.active}</div>
+                        </div>
+                        <div>
+                            <div className={styles.statHeading}>Avg rev member / month</div>
+                            <div className={styles.statValue}>
+                                {Math.floor(revenuePerMonthPerMember)}
+                                <span className={styles.currency}> DATA</span>
+                            </div>
+                        </div>
+                        <div>
+                            <div className={styles.statHeading}>Subscribers</div>
+                            <div className={styles.statValue}>{subscriberCount}</div>
+                        </div>
+                        <div>
+                            <div className={styles.statHeading}>Admin fee</div>
+                            <div className={styles.statValue}>{adminFee}</div>
+                        </div>
+                        <div>
+                            <div className={styles.statHeading}>Product created</div>
+                            <div className={styles.statValue}>{product && product.created && new Date(product.created).toLocaleDateString()}</div>
+                        </div>
+                    </div>
+                    <div className={styles.graphs}>
+                        <div className={styles.memberContainer}>
+                            <div className={styles.memberHeadingContainer}>
+                                <div className={styles.statHeading}>Members</div>
+                                <Dropdown
+                                    title=""
+                                    selectedItem={shownDays.toString()}
+                                    onChange={(item) => setShownDays(Number(item))}
+                                    className={styles.memberGraphDropdown}
+                                    toggleStyle="small"
+                                >
+                                    <Dropdown.Item value="7">Last 7 days</Dropdown.Item>
+                                    <Dropdown.Item value="28">Last 28 days</Dropdown.Item>
+                                    <Dropdown.Item value="90">Last 90 days</Dropdown.Item>
+                                </Dropdown>
+                            </div>
+                            <MembersGraph
+                                className={styles.graph}
+                                authApiKeyId={authApiKeyId}
+                                joinPartStreamId="DRaCVLn1TOWpe7I76gN8hA"
```

Comment:
This should probably be an environment variable since it will change depending on the environment this is running in, right? Actually, where does this come from?

Reply:
Ah, this is test data that works only on my machine. This would be replaced by value read from the community product smart contract. I'll fix this when we have all the necessary pieces from https://github.com/streamr-dev/streamr-platform/pull/696 👍